### PR TITLE
Update problem.py

### PR DESCRIPTION
**Pull request by tschm**
Adding variable and parameter dict

### CODE REVIEW

File: cvxpy/problems/problem.py [hunk 1]
```diff
@@ -356,6 +356,14 @@ def solver_stats(self):
         """
         return self._solver_stats
 
+    @property
+    def parameters(self):
+        return {parameter.name(): parameter for parameter in self.problem.parameters()}
```
Comment (SteveDiamond) [inline cvxpy/problems/problem.py:361]: Could you change these methods to "param_dict" and "var_dict"? We already have parameters and variables methods, and we can't break that interface.

**Comment by SteveDiamond** [comment]
Thanks for doing a PR! We can walk you through the process, don't worry :) You will need to add tests though. Just make a new commit to the fork you're basing the PR from.

**Comment by tschm** [comment]
Sorry, I can't open appveyor because of an outdated security configuration on your side?

**Comment by SteveDiamond** [comment]
I'm closing this since you moved things to a different PR.

**Pull request closed without merging**